admin_guide/pruning_resources.adoc

```diff
@@ -615,3 +615,13 @@ finished, the registry can be switched back to read-write mode by executing:
 $ oc env -n default dc/docker-registry REGISTRY_STORAGE_MAINTENANCE_READONLY-
 ----
 endif::[]
+
+[[pruning-cronjobs]]
+== Pruning Cron Jobs
+Cron Jobs are a technology preview feature, this means it lacks some capabilities to be
+full first class cluster citizen.  One of the biggest issue is that cron jobs do not
+perform automatic pruning of the jobs it creates.  For that reason, cluster administrator
+should perform regular xref:../dev_guide/cron_jobs.adoc#cleaning-up-after-a-cron-job[cleanup of jobs],
+manually.  Additionally, it is advisable to xref:manage_authorization_policy.adoc[restrict the access]
+to it to a small group of trusted users and set appropriate xref:quota.adoc[quota] to prevent
+cron job from creating too many jobs and pods.
```

Comment:
"Cron Jobs are a technology preview feature, this means it lacks some capabilities to be full first class cluster citizen.  One of the biggest issue is that cron jobs do not perform automatic pruning of the jobs it creates.  For that reason, cluster administrator should perform regular xref:../dev_guide/cron_jobs.adoc#cleaning-up-after-a-cron-job[cleanup of jobs], manually.  Additionally, it is advisable to xref:manage_authorization_policy.adoc[restrict the access] to it to a small group of trusted users and set appropriate xref:quota.adoc[quota] to prevent cron job from creating too many jobs and pods."

Should be:

"Cron jobs are a Technology Preview feature only.  Technology Preview features might not be functionally complete. These features provide early access to upcoming product features, enabling users to test functionality and provide feedback during the development process. At this time, the results of cron jobs are not automatically pruned.  The cluster administrator should manually xref:../dev_guide/cron_jobs.adoc#cleaning-up-after-a-cron-job[clean up cron job results] on a regular basis.  Additionally, it is advisable to xref:manage_authorization_policy.adoc[restrict access] to this feature to a small group of trusted users and set a xref:quota.adoc[quota] to prevent cron jobs from creating too many jobs or pods."